### PR TITLE
Use classList.toggle in dark-mode.mdx example

### PR DESCRIPTION
**Pull request by imme-emosol**
Use classList.toggle in dark-mode.mdx example
Support : https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/toggle , https://developer.mozilla.org/en-US/docs/Web/API/Element/classList .

### CODE REVIEW

File: src/pages/docs/dark-mode.mdx [hunk 1]
```diff
@@ -139,11 +139,10 @@ Here's a simple example of how you can support light mode, dark mode, as well as
 
 ```js {{ filename: 'spaghetti.js' }}
 // On page load or when changing themes, best to add inline in `head` to avoid FOUC
-if (localStorage.theme === 'dark' || (!('theme' in localStorage) && window.matchMedia('(prefers-color-scheme: dark)').matches)) {
-  document.documentElement.classList.add('dark')
-} else {
-  document.documentElement.classList.remove('dark')
-}
+document.documentElement.classList.remove(
```
Comment (adamwathan) [inline src/pages/docs/dark-mode.mdx:142]: This should be `toggle` right?

Comment (imme-emosol) [reply src/pages/docs/dark-mode.mdx:142]: d0h! yes. should i force push a new commit with that fix?

Comment (imme-emosol) [reply src/pages/docs/dark-mode.mdx]: wow.
maybe i drank too much #C0FF33 .

**Comment by adamwathan** [comment]
Haven't tested but I'm going to trust you, don't let me down! 😄 Thanks!

**Pull request merged**